src/sentry/lang/native/plugin.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, *args, **kwargs):
         # The (iOS) symbolserver is still used regardless of this value.
         self.use_symbolicator = _is_symbolicator_enabled(self.project, self.data)
 
+        metrics.incr('native.use_symbolicator', tags={'value': self.use_symbolicator})
```

Comment:
That might now fire even if the processor doesn't run, but I'm fine with that.

Reply:
The processor doesn't run if it's a minidump, but we still want the sampling decision.

I think we should be fine if we see that this roughly matches what we set in options. Otherwise we can also add a third call to `_should_use_...` just to get the metric exactly once per event.

Reply:
Wonder if a safer place would be the enhancer then. Also, beware that `RetryProcessing` will make this fire multiple times per event. But all of that is fine given that we just care how often the job is run.

Reply:
Good point about `RetryProcessing`, I didn't even consider that. I think we will see if it skews the metrics too much. Am playing around with not sending the metric if we see a request ID in redis, and I don't like how that looks.